harness/determined/common/experimental/checkpoint/_checkpoint.py

```diff
@@ -49,6 +49,34 @@ class CheckpointState(enum.Enum):
     PARTIALLY_DELETED = bindings.checkpointv1State.PARTIALLY_DELETED.value
 
 
+class CheckpointOrderBy(enum.Enum):
+    """
+    Specifies whether a sorted list of checkpoints should be in ascending or
+    descending order.
+    """
+
+    ASC = bindings.v1OrderBy.ASC.value
+    DESC = bindings.v1OrderBy.DESC.value
+
+    def _to_bindings(self) -> bindings.v1OrderBy:
+        return bindings.v1OrderBy(self.value)
+
+
+class CheckpointSortBy(enum.Enum):
+    """Specifies checkpoint parameters that can be used for sorting checkpoints."""
+
+    UNSPECIFIED = bindings.checkpointv1SortBy.UNSPECIFIED
+    UUID = bindings.checkpointv1SortBy.UUID
```

Comment:
> It seems more right to me for the value of a CheckpointSortBy instance to be a bindings.checkpointv1SortBy value rather than a bindings.checkpointv1SortBy instance

yeah, this makes sense. updated.